modules/ghlint/src/test/kotlin/net/twisterrob/ghlint/GHLintTest.kt

```diff
@@ -385,7 +385,8 @@ class GHLintTest {
 				    name: "Test"
 				    timeout-minutes: 10
 				    runs-on: ubuntu-latest
-				    permissions: {}
+				    permissions:
+				      contents: read
```

Comment:
look, the rule found a missing permission in a test :)

(this commit is to update from main after a big refactor in testing syntax)